index.js

```diff
@@ -0,0 +1,120 @@
+const hamburger = document.getElementById('menu');
+const closeBtn = document.getElementById('close');
+const menuItem = document.getElementById('item');
+const li = document.getElementById('li');
+const items = document.querySelectorAll('.item');
+const Spcan = document.getElementById('Spcann');
+
+items.forEach((elmt) => {
+  elmt.addEventListener('click', () => {
+    menuItem.style = 'display: none;';
+  });
+});
+
+hamburger.addEventListener('click', () => {
+  menuItem.style = 'display: block;';
+});
+
+closeBtn.addEventListener('click', () => {
+  menuItem.style = 'display: none;';
+});
+
+li.addEventListener('click', () => {
+  menuItem.style = 'display: none;';
+});
+
+const Speakers = [
+  {
+    name: 'Neymar Jr',
+    image: './Images/NeymarJr.jpg',
+    discrip: 'He is a professional footballer who has a lot of skills in his career of life',
+    title: 'Fulltime professional footballer',
+  },
+  {
+    name: 'Jack Dorsey',
+    image: './Images/CEOtwitter.jpg',
+    discrip: 'The founder and CEO of the well known social media app Twitter.',
+    title: 'CEO of Twitter',
+  },
+  {
+    name: 'Jeff Bezos',
+    image: './Images/Bezos.jpg',
+    discrip: 'A strong business man who succeeded in his own business',
+    title: 'One of the richest men in the world presently',
+  },
+  {
+    name: 'Bill Gate',
+    image: './Images/BillGate.jpg',
+    discrip: 'The founder of the Windows Operating System which has developed the world we live in today.',
+    title: 'Founder of Windows and a rich man too.',
+  },
+  {
+    name: 'Elon Musk',
+    image: './Images/ElonMusk.jpg',
+    discrip: 'He owns one of the best electric motor and airship companies, Tesla and Space-X',
+    title: 'CEO of Tesla and Space-X',
+  },
+  {
+    name: 'Mark Zuckerberg',
+    image: './Images/Mark.jpg',
+    discrip: 'The founder and CEO of the well known social media app Facebook.',
+    title: 'CEO of facebook platform',
+  },
+];
```

Comment:
- [ ] [OPTIONAL] Nice work dynamically generating feature speaker with javascript arrays but i notice that the speakers are not rendering on you page kindly check it out